### PR TITLE
"Connection is already acquired" error

### PR DESCRIPTION
**Pull request by bokolob**

### CODE REVIEW

**Comment by bokolob** [comment]
I've written a new version of the tests, and they reproduce the error. But, the suit hangs up after mysql/postgres tests finish. No idea why.

File: databases/backends/mysql.py [hunk 1]
```diff
@@ -97,9 +97,12 @@ async def acquire(self) -> None:
     async def release(self) -> None:
         assert self._connection is not None, "Connection is not acquired"
         assert self._database._pool is not None, "DatabaseBackend is not running"
-        await self._database._pool.release(self._connection)
+        await self._internal_release(self._connection)
```
Comment (aminalaee) [inline databases/backends/mysql.py:100]: Sorry I'm confused, how is this different from the original one?

File: databases/backends/postgres.py [hunk 2]
```diff
@@ -263,7 +267,7 @@ def _compile(self, query: ClauseElement) -> typing.Tuple[str, list, tuple]:
 
     @staticmethod
     def _create_column_maps(
-        result_columns: tuple,
+            result_columns: tuple,
```
Comment (aminalaee) [inline databases/backends/postgres.py:270]: I think it's mixing tab and spaces in multiple files. You can use `./scripts/lint` to fix that.

File: databases/backends/mysql.py (hunk 1 above)
Comment (bokolob) [reply databases/backends/mysql.py:100]: It seems, I forgot to make changes for mysql during debugging

**Comment by aminalaee** [comment]
Any update with this?

**Comment by aminalaee** [comment]
I think this is out of date now. Closing it.

**Comment by bokolob** [comment]
Hello, no it’s still in progress. I haven’t had enough time for this. So you can help with tests :)

**Comment by Beaend** [comment]
i got this error then write " instead of '
`SELECT {need} FROM {table_name} WHERE {column_name} = "{column_text}"`
first i got error `UndefinedColumnError`, then this.
(i'm use pool)

**Pull request closed without merging**